Networking/NetworkingTests/Remote/BlazeRemoteTests.swift

```diff
@@ -431,6 +431,48 @@ final class BlazeRemoteTests: XCTestCase {
         XCTAssertEqual(result, .init(totalImpressionsMin: 17900, totalImpressionsMax: 24200))
     }
 
+    func test_fetchForecastedImpressions_correct_parameters() async throws {
```

Comment:
Thanks for adding this test. 
```suggestion
    func test_fetchForecastedImpressions_sends_correct_parameters() async throws {
```